src/asm_cfg.cpp

```diff
@@ -88,6 +90,7 @@ static Condition::Op reverse(Condition::Op op) {
     return {};
 }
 
+/// Get the inverse of a given comparison condition.
 static Condition reverse(Condition cond) { return {.op = reverse(cond.op), .left = cond.left, .right = cond.right}; }
```

Comment:
Perhaps we should rename these functions to `invert_comparison` if that is not clear.

Reply:
That would be clearer.  Although this PR is so far only about comments.  If we want to do new renames in here too I can do that, let me know.  Otherwise could do renames in a subsequent PR.

Reply:
Better not to mix. You are right.